Rectangle roi
geometry_msgs/Point32[4] table_corners
Circle[] ball_trajectory
Circle[] predicted_ball_trajectory
sensor_msgs/Image ball_image
